setup.py

```diff
@@ -1,19 +1,32 @@
 """Install nix-bisect"""
 
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(
     name="nix-bisect",
     version="0.4.1",
     description="Bisect nix builds",
     author="Timo Kaufmann",
-    packages=["nix_bisect"],
+    include_package_data=True,
+    packages=find_packages(),
+    py_modules = [
+        'bisect_env',
+        'bisect_runner',
+        'build_status',
+        'derivation',
+        'exceptions',
+        'extra_bisect',
+        'gcroot',
+        'git',
+        'git_bisect',
+        'nix',
+        'test_util',
+    ],
```

Comment:
Why is this necessary?

Reply:
It wasn't, again my mistake, sorry. Fixed.